spinnman/processes/get_machine_process.py

```diff
@@ -139,6 +151,8 @@ def _receive_error(self, request, exception, tb):
 
     def get_machine_details(self, boot_x, boot_y, width, height,
                             repair_machine, ignore_bad_ethernets):
+
+        machine = machine_from_size(width, height)
```

Comment:
its built but not used till right at the end of this function. build at the bottom for better readability

Reply:
Ok delayed until just before the ignore process methods as they need the machine to handle wrap arounds.